src/palace/manager/api/s3_analytics_provider.py

```diff
@@ -25,13 +26,14 @@ def __init__(self, s3_service: S3Service | None):
     @staticmethod
     def _create_event_object(
         library: Library,
-        license_pool: LicensePool,
+        license_pool: LicensePool | None,
         event_type: str,
         time: datetime.datetime,
-        old_value,
-        new_value,
+        old_value: Any = None,
+        new_value: Any = None,
```

Comment:
Minor: It seems like we can probably do better for a type annotation then `Any` here. I really try to avoid `Any` as much as possible. It seems like these are probably `int | None` right?